cookbooks/swift/templates/default/etc/swift/proxy-server/proxy-server.conf.d/20_settings.conf.erb

```diff
@@ -21,3 +21,16 @@ user_test2_tester2 = testing2 .admin
 
 [filter:nvratelimit]
 use = egg:nvratelimit#nvratelimit
+
+[filter:nvauth]
+use = egg:nvauth_middleware#nvauth_middleware
+
+[filter:nvsts]
+use = egg:nvsts_middleware#nvsts_middleware
+jwks_endpoint_1 = https://maglev-dev-sjc4.nvda.ai/authn/storage/.well-known/jwks.json
+jwks_issuer_1 = maglev-stage
+jwks_ttl_1 = 3600000
+jwks_get_groups_1 = maglev
+jwks_aud_1 = swiftstack
+jwks_roles_endpoint_1 = https://example.api.nvidia.com/roles
+jwks_roles_ttl_1 = 60000
```

Comment:
these could just be extra files in conf.d - maybe we'd have to support extra cookbooks - but MAYBE maybe we could just support a `BASE_ETC_SWIFT = ` that pre-create some files/directories to overlay into `/etc/swift/`